markets/bfp-market/contracts/modules/SplitAccountConfigurationModule.sol

```diff
@@ -0,0 +1,61 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.11 <0.9.0;
+
+import {IERC165} from "@synthetixio/core-contracts/contracts/interfaces/IERC165.sol";
+import {OwnableStorage} from "@synthetixio/core-contracts/contracts/ownership/OwnableStorage.sol";
+import {ERC165Helper} from "@synthetixio/core-contracts/contracts/utils/ERC165Helper.sol";
+import {ERC2771Context} from "@synthetixio/core-contracts/contracts/utils/ERC2771Context.sol";
+import {ISplitAccountConfigurationModule} from "../interfaces/ISplitAccountConfigurationModule.sol";
+import {SplitAccountConfiguration} from "../storage/SplitAccountConfiguration.sol";
+import {ErrorUtil} from "../utils/ErrorUtil.sol";
+
+contract SplitAccountConfigurationModule is ISplitAccountConfigurationModule {
+    // --- Mutations --- //
+
+    /**
+     * @inheritdoc ISplitAccountConfigurationModule
+     */
+    function setEndorsedSplitAccount(address[] memory addresses) external {
+        OwnableStorage.onlyOwner();
+
+        SplitAccountConfiguration.GlobalData storage config = SplitAccountConfiguration.load();
+        uint256 existingAddresses = config.whitelistedAddresses.length;
+        for (uint256 i = 0; i < existingAddresses; ) {
+            delete config.whitelisted[config.whitelistedAddresses[i]];
+            unchecked {
+                ++i;
+            }
+        }
+        delete config.whitelistedAddresses;
+
+        uint256 addressesLength = addresses.length;
+
+        for (uint256 i = 0; i < addressesLength; ) {
```

Comment:
Should add some minimal checks like not allowing zero addresses.

Reply:
Added 0x address check